src/sentry/consumers/__init__.py

```diff
@@ -259,6 +259,7 @@ def ingest_events_options() -> list[click.Option]:
         "static_args": {
             "consumer_type": "events",
         },
+        "dlq_topic": Topic.INGEST_EVENTS_DLQ,
```

Comment:
Has the topic already been created in all environments ?

Reply:
yes